console/console-init/ui/src/graphql-module/queries/iot_device.ts

```diff
@@ -44,4 +46,87 @@ const DELETE_IOT_DEVICE = gql`
   }
 `;
 
-export { RETURN_IOT_DEVICE_DETAIL, RETURN_IOT_CREDENTIALS, DELETE_IOT_DEVICE };
+const RETURN_ALL_DEVICES_FOR_IOT_PROJECT_SORT = (sortBy?: ISortBy) => {
```

Comment:
I feel prefix should be SORT and FILTER which sounds more good. can we try.
SORT_RETURN_ALL_DEVICES_FOR_IOT_PROJECT
FILTER_RETURN_ALL_DEVICES_FOR_IOT_PROJECT